eyes_selenium/applitools/selenium/visual_grid/render_task.py

```diff
@@ -214,53 +215,66 @@ def parse_frame_dom_resources(self, data):  # noqa
                 frames_num=len(frames),
             )
         )
-        frame_request_resources = {}
-        discovered_resources_urls = set()
 
-        def handle_resources(content_type, content, resource_url):
+        def find_child_resource_urls(content_type, content, resource_url):
             # type: (Optional[Text], bytes, Text) -> NoReturn
             logger.debug(
-                "handle_resources({0}, {1}) call".format(content_type, resource_url)
+                "find_child_resource_urls({0}, {1}) call".format(
+                    content_type, resource_url
+                )
             )
             if not content_type:
```

Comment:
Check if there's a way to collect resources even if content type is not provided. (TBD)